modules/oe_whitelabel_list_pages/tests/src/Functional/ListPagesTest.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_whitelabel_list_pages_test\FunctionalJavascript;
```

Comment:
This is the wrong namespace..
I know this PR is merged, but I don't know another quick and easy way to mention it.
We can fix it some day in the future when we touch this class again.